### ISSUE TITLE
Pixellated emojis on Firefox on Windows 

### ISSUE THREAD

**Issue by Salamandar** [vector-im/element-web#16588]
### Description

The emojis are badly pixellated on Firefox on Windows.

It's OK on Firefox on Linux, it's OK on Edge.

The issue was also present for the round buttons (go to first unread message, etc) but it recently became better.

Here is how the emoji panel looks like on Firefox 86 on Windows : 

![image](https://user-images.githubusercontent.com/6552989/109780807-a3397880-7c07-11eb-9544-268baea7a61d.png)

Here is how it looks like on Edge :

![image](https://user-images.githubusercontent.com/6552989/109780870-b0eefe00-7c07-11eb-8654-f9376eb1d919.png)

**Comment by yisibl** [comment]
Please consider using nanoemoji to generate COLR fonts https://github.com/googlefonts/nanoemoji/

**Comment by jryans** [comment]
@Salamandar, I don't think I can reproduce this issue... Here's a screenshot from Firefox 89 on Windows 10 20H2:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/279572/112861982-36ea5180-90ad-11eb-8c40-914bef707c6f.png">

We already have COLR fonts, and we specifically use them on Firefox. Is this a self-hosted deployment? Please send debug logs, and compare with https://develop.element.io's behaviour.

**Comment by Salamandar** [comment]
@jryans You should try on Firefox stable, aka 87.0.

**Comment by jryans** [comment]
@Salamandar Can you send a bit more info about your screen and pixel ratio? What do you get for these values in the console?

```
> screen
Screen { availWidth: 1680, availHeight: 1027, width: 1680, height: 1050, colorDepth: 24, pixelDepth: 24, top: 0, left: 0, availTop: 23, availLeft: 0 }
> devicePixelRatio
2
```

**Comment by Salamandar** [comment]
@jryans here it is !

```
> screen
Screen { availWidth: 1920, availHeight: 1040, width: 1920, height: 1080, colorDepth: 24, pixelDepth: 24, top: 0, left: 0, availTop: 0, availLeft: 0 }

> devicePixelRatio
1
```

**Comment by Salamandar** [comment]
Here is the full log after a full reload of Element :
```
/sync error ConnectionError: request failed: CORS request rejected: https://matrix-client.matrix.org/_matrix/client/r0/sync?filt?15025_804692172_686273230_2184865_237743021_880571634_184720 vendors~init.js:2:1252553
Error: request failed: CORS request rejected: https://matrix-client.matrix.org/_matrix/client/r0/sync?filt?15025_804692172_686273230_2184865_237743021_880571634_184720
    v https://app.element.io/bundles/6421c751c3aa2dc359ed/vendors~init.js:2
    _ https://app.element.io/bundles/6421c751c3aa2dc359ed/vendors~init.js:2
    g https://app.element.io/bundles/6421c751c3aa2dc359ed/vendors~init.js:2
    u https://app.element.io/bundles/6421c751c3aa2dc359ed/vendors~init.js:2
    onreadystatechange https://app.element.io/bundles/6421c751c3aa2dc359ed/vendors~init.js:2
vendors~init.js:2:1252553
Number of consecutive failed sync requests: 1 vendors~init.js:2:1252553
Starting keep-alive vendors~init.js:2:1252553
MatrixClient sync state => RECONNECTING vendors~init.js:2:1252553
Initialised rageshake. KeyboardShortcuts.tsx:280:25
To fix line numbers in Chrome: Meatball menu ? Settings ? Blackboxing ? Add /rageshake\.js$ KeyboardShortcuts.tsx:280:25
Using Web platform KeyboardShortcuts.tsx:280:25
XHRGEThttps://app.element.io/config.app.element.io.json?cachebuster=1617180901247
[HTTP/2 404 Not Found 70ms]

Configuring rageshake persistence... KeyboardShortcuts.tsx:280:25
Loading skin... KeyboardShortcuts.tsx:280:25
onmozfullscreenchange est obsol?te. velocity.min.js:6:36577
onmozfullscreenerror est obsol?te. velocity.min.js:6:36577
Skin loaded! KeyboardShortcuts.tsx:280:25
Using WebAssembly Olm KeyboardShortcuts.tsx:280:25
Content Security Policy: Les param?tres de la page ont emp?ch? le chargement d?une ressource ? blob:https://app.element.io/9a805b8b-99b5-4053-9306-3911fdae52c3 (? worker-src ?).
set language to fr KeyboardShortcuts.tsx:280:25
Vector starting at https://app.element.io/ KeyboardShortcuts.tsx:280:25
Verifying homeserver configuration KeyboardShortcuts.tsx:280:25
Config uses a default_server_name - doing .well-known lookup KeyboardShortcuts.tsx:280:25
DEPRECATED CONFIG OPTION: In the future, default_server_name will not be accepted. Please use default_server_config instead. KeyboardShortcuts.tsx:280:25
Erreur d?analyse XML : mal form?
Emplacement : https://matrix.org/.well-known/matrix/client
Num?ro de ligne 1, Colonne 1 : client:1:1
Using homeserver config: 
Object { hsUrl: "https://matrix-client.matrix.org", hsName: "matrix.org", hsNameIsDifferent: true, isUrl: "https://vector.im", isDefault: true, isNameResolvable: true, warning: null }
KeyboardShortcuts.tsx:280:25
Updating SdkConfig with validated discovery information KeyboardShortcuts.tsx:280:25
No pickle key available KeyboardShortcuts.tsx:280:25
Restoring session for @Salamandar:matrix.org KeyboardShortcuts.tsx:280:25
setLoggedIn: mxid: @Salamandar:matrix.org deviceId: TBJOQEPCFH guest: false hs: https://matrix-client.matrix.org/ softLogout: false  freshLogin: false KeyboardShortcuts.tsx:280:25
StorageManager: Checking storage consistency KeyboardShortcuts.tsx:280:25
StorageManager: Local storage supported? true KeyboardShortcuts.tsx:280:25
StorageManager: IndexedDB supported? true KeyboardShortcuts.tsx:280:25
StorageManager: Local storage contains data? true KeyboardShortcuts.tsx:280:25
StorageManager: Crypto initialised? true KeyboardShortcuts.tsx:280:25
StorageManager: Sync store using IndexedDB contains data? true KeyboardShortcuts.tsx:280:25
StorageManager: Crypto store using IndexedDB contains data? true KeyboardShortcuts.tsx:280:25
StorageManager: Storage consistency checks passed KeyboardShortcuts.tsx:280:25
Session persisted for @Salamandar:matrix.org KeyboardShortcuts.tsx:280:25
Lifecycle: Starting MatrixClient KeyboardShortcuts.tsx:280:25
EventIndex: Platform doesn't support event indexing, not initializing. KeyboardShortcuts.tsx:280:25
IndexedDBStore.startup: connecting to backend KeyboardShortcuts.tsx:280:25
MatrixClientPeg: waiting for MatrixClient store to initialise KeyboardShortcuts.tsx:280:25
Switching to room id !qHKbDnFNmMBVMAiVgX:matrix.org at event undefined KeyboardShortcuts.tsx:280:25
StorageManager: Persistent? true KeyboardShortcuts.tsx:280:25
IndexedDB worker is ready KeyboardShortcuts.tsx:280:25
LocalIndexedDBStoreBackend.connect: connecting... indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend.connect: awaiting connection... indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend.connect: connected indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend: loading account data... indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend: loading sync data... indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend: loaded account data indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend: loaded sync data indexeddb-worker.js:1:1171
LocalIndexedDBStoreBackend: loaded initial data indexeddb-worker.js:1:1171
IndexedDBStore.startup: loading presence events KeyboardShortcuts.tsx:280:25
IndexedDBStore.startup: processing presence events KeyboardShortcuts.tsx:280:25
Crypto: Starting up crypto store... KeyboardShortcuts.tsx:280:25
connecting to indexeddb matrix-js-sdk:crypto KeyboardShortcuts.tsx:280:25
connected to indexeddb matrix-js-sdk:crypto KeyboardShortcuts.tsx:280:25
Crypto: initialising roomlist... KeyboardShortcuts.tsx:280:25
Crypto: initialising crypto object... KeyboardShortcuts.tsx:280:25
Crypto: initialising Olm... KeyboardShortcuts.tsx:280:25
Crypto: initialising Olm device... KeyboardShortcuts.tsx:280:25
Crypto: loading device list... KeyboardShortcuts.tsx:280:25
Crypto: fetching own devices... KeyboardShortcuts.tsx:280:25
Loaded cross-signing public keys from crypto store KeyboardShortcuts.tsx:280:25
Crypto: checking for key backup... KeyboardShortcuts.tsx:280:25
Checking key backup status... KeyboardShortcuts.tsx:280:25
MatrixClientPeg: really starting MatrixClient KeyboardShortcuts.tsx:280:25
Fetching new TURN credentials KeyboardShortcuts.tsx:280:25
Getting saved sync token... KeyboardShortcuts.tsx:280:25
Getting push rules... KeyboardShortcuts.tsx:280:25
MatrixClientPeg: MatrixClient started KeyboardShortcuts.tsx:280:25
Attempting to get Jitsi conference information from homeserver KeyboardShortcuts.tsx:280:25
Jitsi conference domain: jitsi.riot.im KeyboardShortcuts.tsx:280:25
Backup public key l8gE0kueFKyLnsV3+zkaSnke+ZoeZQIcgR5X5fKWXjs is trusted locally KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:WILMPPWDKA KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:VURBRLSTAK KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:VJUZDGJSVB KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:KYTTKOUHUW KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:PHASVEZXTX KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:UGANRGQSCC KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:TAMMGRRGLC KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:AYWXQVOVCX KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:GTRKXLVXTK KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:SHRGLKQYPE KeyboardShortcuts.tsx:280:25
Ignoring signature from unknown key ed25519:TJUOFQCYJF KeyboardShortcuts.tsx:280:25
Found usable key backup v5: enabling key backups KeyboardShortcuts.tsx:280:25
Got saved sync token KeyboardShortcuts.tsx:280:25
Getting saved sync... KeyboardShortcuts.tsx:280:25
Got reply from saved sync, exists? true KeyboardShortcuts.tsx:280:25
sync(): not doing HTTP hit, instead returning stored /sync data KeyboardShortcuts.tsx:280:25
returning explicit theme: dark KeyboardShortcuts.tsx:280:25
Enabling encryption in !hXIvOoHlzTNXNCpRhu:matrix.org; starting to track device lists for all users therein KeyboardShortcuts.tsx:280:25
Starting to track devices for room !hXIvOoHlzTNXNCpRhu:matrix.org ... KeyboardShortcuts.tsx:280:25
LL: got 2 membershipEvents from storage for room !hXIvOoHlzTNXNCpRhu:matrix.org ... indexeddb-worker.js:1:1171
Updating homeserver-configured integration managers... KeyboardShortcuts.tsx:280:25
Homeserver has no integration managers KeyboardShortcuts.tsx:280:25
Attempting to get Jitsi conference information from homeserver KeyboardShortcuts.tsx:280:25
Erreur d?analyse XML : mal form?
Emplacement : https://matrix.org/.well-known/matrix/client
Num?ro de ligne 1, Colonne 1 : client:1:1
Jitsi conference domain: jitsi.riot.im KeyboardShortcuts.tsx:280:25
LL: RoomState about to set 2 OOB members ... KeyboardShortcuts.tsx:280:25
LL: RoomState put in OOB_STATUS_FINISHED state ... KeyboardShortcuts.tsx:280:25
Join event for @yannsionneau:matrix.org in !hXIvOoHlzTNXNCpRhu:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !YbqwDBgmEZDcqYPJgk:matrix.org; starting to track device lists for all users therein KeyboardShortcuts.tsx:280:25
Starting to track devices for room !YbqwDBgmEZDcqYPJgk:matrix.org ... KeyboardShortcuts.tsx:280:25
LL: got 0 membershipEvents from storage for room !YbqwDBgmEZDcqYPJgk:matrix.org ... indexeddb-worker.js:1:1171
Got push rules KeyboardShortcuts.tsx:280:25
Checking lazy load status... KeyboardShortcuts.tsx:280:25
Checking server lazy load support... KeyboardShortcuts.tsx:280:25
Enabling lazy load on sync filter... KeyboardShortcuts.tsx:280:25
Checking whether lazy loading has changed in store... KeyboardShortcuts.tsx:280:25
Got TURN URIs: turn:turn.matrix.org:3478?transport=udp,turn:turn.matrix.org:3478?transport=tcp,turns:turn.matrix.org:443?transport=tcp refresh in 86400 secs KeyboardShortcuts.tsx:280:25
LL: RoomState about to set 0 OOB members ... KeyboardShortcuts.tsx:280:25
LL: RoomState put in OOB_STATUS_FINISHED state ... KeyboardShortcuts.tsx:280:25
Error decrypting event (id=$GHFoxFpLAaPEwe1nr8apkPh_ljqQUFHoBAmTaF8uze8): DecryptionError[msg: Unknown encryption algorithm "undefined".] KeyboardShortcuts.tsx:280:25
Enabling encryption in !fMXOEvQhENrjDIdDGk:matrix.org; starting to track device lists for all users therein KeyboardShortcuts.tsx:280:25
Starting to track devices for room !fMXOEvQhENrjDIdDGk:matrix.org ... KeyboardShortcuts.tsx:280:25
LL: got 0 membershipEvents from storage for room !fMXOEvQhENrjDIdDGk:matrix.org ... indexeddb-worker.js:1:1171
LL: RoomState about to set 0 OOB members ... KeyboardShortcuts.tsx:280:25
LL: RoomState put in OOB_STATUS_FINISHED state ... KeyboardShortcuts.tsx:280:25
Enabling encryption in !fDqaHgtMTHvsuoQOay:matrix.org; starting to track device lists for all users therein KeyboardShortcuts.tsx:280:25
Starting to track devices for room !fDqaHgtMTHvsuoQOay:matrix.org ... KeyboardShortcuts.tsx:280:25
Storing client options... KeyboardShortcuts.tsx:280:25
LL: got 0 membershipEvents from storage for room !fDqaHgtMTHvsuoQOay:matrix.org ... indexeddb-worker.js:1:1171
LL: RoomState about to set 0 OOB members ... KeyboardShortcuts.tsx:280:25
LL: RoomState put in OOB_STATUS_FINISHED state ... KeyboardShortcuts.tsx:280:25
Enabling encryption in !KLaToumxJaYCDrtgdX:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !jNPodAIexjWLNhgGsD:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !plLrGkDQuQJYCHjkzo:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !wKsPfPLfOgyAJFmfbU:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !qHKbDnFNmMBVMAiVgX:matrix.org KeyboardShortcuts.tsx:280:25
Enabling encryption in !pmbQytUUFxVFRHYQYp:matrix.org KeyboardShortcuts.tsx:280:25
MatrixClient sync state => PREPARED KeyboardShortcuts.tsx:280:25
Checking for COLR support KeyboardShortcuts.tsx:280:25
Browser is Firefox - assuming COLR is supported KeyboardShortcuts.tsx:280:25
RVS update: !qHKbDnFNmMBVMAiVgX:matrix.org undefined loading? false joining? false initial? true shouldPeek? true KeyboardShortcuts.tsx:280:25
newscreen room/!qHKbDnFNmMBVMAiVgX:matrix.org KeyboardShortcuts.tsx:280:25
updateTint from RoomView.gatherTimelinePanelRef KeyboardShortcuts.tsx:280:25
Tinter.tint from updateTint KeyboardShortcuts.tsx:280:25
LL: got 0 membershipEvents from storage for room !qHKbDnFNmMBVMAiVgX:matrix.org ... indexeddb-worker.js:1:1171
Stored client options KeyboardShortcuts.tsx:280:25
Getting filter... KeyboardShortcuts.tsx:280:25
Setting up Mjolnir: after sync KeyboardShortcuts.tsx:280:25
Updating Mjolnir ban lists to: KeyboardShortcuts.tsx:280:25
LL: RoomState about to set 0 OOB members ... KeyboardShortcuts.tsx:280:25
LL: RoomState put in OOB_STATUS_FINISHED state ... KeyboardShortcuts.tsx:280:25
Starting to track devices for room !qHKbDnFNmMBVMAiVgX:matrix.org ... KeyboardShortcuts.tsx:280:25
Caching capabilities:  
Object { "m.room_versions": {?}, "m.change_password": {?} }
KeyboardShortcuts.tsx:280:25
[!qHKbDnFNmMBVMAiVgX:matrix.org] Current version: 5 KeyboardShortcuts.tsx:280:25
[!qHKbDnFNmMBVMAiVgX:matrix.org] Version capability:  
Object { default: "6", available: {?} }
KeyboardShortcuts.tsx:280:25
Sending first sync request... KeyboardShortcuts.tsx:280:25
Waiting for saved sync before starting sync processing... KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.key event with id:$Vvtrpv7sTjWcab9eEN742W3JS92XSa3THaEPUhIYCyQ, content:{"key":"Gne5pF3unCkmeFT1wJ8l8BMQsV9zXcRvcfCOQq37ZWM"} deviceId:undefined, sender:@Salamandar:matrix.org, isSentByUs:true, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:false=>true KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.key event with id:$9zpY4T7uyehmK9aoTMwxa4NCA0pF7X7gz3wjo2TPxhI, content:{"key":"sidzw2r2UhVxqVMAnJctOnDD3DsydmxxygVFewMtODg"} deviceId:undefined, sender:@yannsionneau:matrix.org, isSentByUs:false, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.mac event with id:$wIN1W44-JXjIXKhIggCOtnBd5gjedWUGWpHjMzghMLk, content:{"mac":{"ed25519:CYJBOHGOMP":"OpnbYkZwWnccbmNjYm1OalltMU9hbGx0TVU5aGJHeDA","ed25519:j+J9iJkUCuZJT6012k2dxIATAmjBBT1Ng0pKjqM2XZ8":"9HEeZVWmV22lMjJsTWpKc1RXcEtjMVJYY0V0ak1WSlk"},"keys":"U1agZ3PtUHSJSFNKU0ZOS1UwWk9TMVV3V2s5VE1WVjM"} deviceId:undefined, sender:@yannsionneau:matrix.org, isSentByUs:false, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.mac event with id:$r88O7Bta9Vxwstr-V_4uaL2cXwWObWsFTRcUHcQGf3s, content:{"keys":"gyAQUUojb2oOMm9PTW05UFRXMDVVRlJYTURWVlJsSlk","mac":{"ed25519:KYTTKOUHUW":"/W0KS4fCZkP7a1A3YTFBM1lURkJNMWxVUmtKTk1XeFY","ed25519:TiR+tvNqp5HyMc2w8Gx8tGjnl6iw8PnwfO4dBcRsCG8":"pwOKS7f2ZjKFaktGYWt0R1lXdDBSMWxYZERCU01XeFk"}} deviceId:undefined, sender:@Salamandar:matrix.org, isSentByUs:true, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.done event with id:$3Pbj4RhDxH7P5fb1Ip3kY0bujtOMHtA4r7aZ-PQ2H1Y, content:{} deviceId:undefined, sender:@yannsionneau:matrix.org, isSentByUs:false, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
MatrixClient sync state => PREPARED KeyboardShortcuts.tsx:280:25
MatrixClient sync state => SYNCING KeyboardShortcuts.tsx:280:25
Regenerating room lists: Startup KeyboardShortcuts.tsx:280:25
Regenerating all room lists KeyboardShortcuts.tsx:280:25
Persisting sync data up to m1914407279~2.1914407285_757284961_3016867_804692926_686274274_2184865_237744007_880574025_184720 indexeddb-worker.js:1:1171
Resetting known rooms, initiating regeneration 2 KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.done event with id:$qo38TlWBbitIc8EyIMqc-ugmDLbUmuatAbqYL0N0eXw, content:{} deviceId:undefined, sender:@Salamandar:matrix.org, isSentByUs:true, isLiveEvent:true, isRemoteEcho:false, phase:1=>1, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
Verification request $G8ZKVlNyybGcRwQJpMotsXsjWzGsG-T3uBVWLo5MTvI: m.key.verification.cancel event with id:$04AgLEmXOBPpu-7lt7GmoK6k8gS6xnWOh8fRhAvWqtU, content:{"code":"m.timeout","reason":"Other party didn't accept in time"} deviceId:undefined, sender:@yannsionneau:matrix.org, isSentByUs:false, isLiveEvent:true, isRemoteEcho:false, phase:1=>5, observeOnly:true=>true KeyboardShortcuts.tsx:280:25
```

**Comment by jryans** [comment]
> > screen
> Screen { availWidth: 1920, availHeight: 1040, width: 1920, height: 1080, colorDepth: 24, pixelDepth: 24, top: 0, left: 0, availTop: 0, availLeft: 0 }
> 
> > devicePixelRatio
> 1

Thanks, so this seems to say you have a low-DPI display. @Salamandar, looking at the comparison screenshots again, the only difference I am noticing is that Firefox on Windows seems to cut off the top and bottom lines, so we should try to fix that... Is there anything else that is different here?

**Comment by Salamandar** [comment]
> Thanks, so this seems to say you have a low-DPI display. @Salamandar, looking at the comparison screenshots again, the only difference I am noticing is that Firefox on Windows seems to cut off the top and bottom lines, so we should try to fix that... Is there anything else that is different here?

Well it's not a low-DPI, just a 24'' 1080p display.

Also, it's not *just* a pixel issue, the font is just not the same. Look at the mouth's shape here : (Linux is up, Windows down). 
![image](https://user-images.githubusercontent.com/6552989/116554433-7948bd80-a8fb-11eb-9ab5-445b90f85177.png)



… Hey ! I just discovered something weird ! When I use the Firefox Inspector, select an emoji and change its display size, it begins being pixellized at 2.04rem !
Here are screenshots at 2.04rem:
![image](https://user-images.githubusercontent.com/6552989/116555979-40114d00-a8fd-11eb-8135-de3acdbdc990.png)

2.05rem:
![image](https://user-images.githubusercontent.com/6552989/116556001-456e9780-a8fd-11eb-8678-91a0b983186f.png)

**Comment by kittykat** [comment]
@Salamandar Are you still seeing this issue with the latest browser versions?
@jryans Could you please help pull out a list of actionable items from this issue?

**Comment by Salamandar** [comment]
Hey @kittykat, i tried again on firefox 92 on Windows, the issue is still present. The emojis are fine on the electron app.

I also just installed firefox nightly 94, the issue is still present.

**Comment by jryans** [comment]
> @jryans Could you please help pull out a list of actionable items from this issue?

I'm afraid I don't yet fully understand the problem, so perhaps someone may need to investigate further first. It appears that at least for this user, when using Firefox on Windows with DPI 1 display, the following problems occur:

* Top and bottom edges of emoji are cut off
* They are pixelated in a somewhat odd way, leading to a worse visual experience

**Comment by jryans** [comment]
I wonder if it's browser or application zoom / font size related...? @Salamandar, are you using either of following?

* Browser page zoom set to some non-default value (something other than 100%)
* Element font size in Settings -> Appearance set to some non-default value (something other than the middle as shown below)

<img width="614" alt="image" src="https://user-images.githubusercontent.com/279572/134157128-e9756552-d234-45d9-a2ad-964b8f9f4e9b.png">

**Comment by frakic** [comment]
Hey @jryans, I've menaged to reproduce this on Windows 11 Home 21H2 by:
1. Turning off the custom font I normally use
2. Making sure my browser's page zoom is set to a default value (100%)
3. Not using a custom font size in Settings

I took a screenshot in two different browsers:
![image](https://user-images.githubusercontent.com/18530109/168585864-22dd06c8-e1d3-4ebb-91db-3de3158de894.png)

As you can see, they're not the same. If we place two emojis next to each other and zoom in really hard, the problem seems to be that Firefox doesn't render some of the low contrast pixels, making the head look cut off and mouth just look different:
![image](https://user-images.githubusercontent.com/18530109/168586887-ca276a03-9a7d-4e6c-a838-7de36aa35556.png)

(if you can't see what I'm describing by zooming in the browser, try downloading the picture and zooming in)

I hope this helps a bit. If there's any additinal info I can give you, just let me know.

**Comment by turt2live** [comment]
I believe the Windows issue will be some variant of https://github.com/vector-im/element-web/issues/22184

For the other issue, I'm presuming this is fixed due to lack of response. If you're still seeing this, please open a new issue.
